lambda/create-case/main.go

```diff
@@ -88,6 +95,15 @@ func (l *Lambda) HandleEvent(event events.APIGatewayProxyRequest) (events.APIGat
 				return ProblemInternalServerError.Respond()
 			}
 
+			measure := event.MeasureOnlineDonor
+			if data.Source == LpaSourcePhone {
+				measure = event.MeasurePaperDonor
+			}
+
+			if err = l.eventClient.SendMetric(ctx, event.CategoryDraftLPACreated, measure); err != nil {
+				l.logger.ErrorContext(ctx, "error sending metric", slog.Any("err", err))
```

Comment:
so in theory we could lose the metric, and the number we get may be less than in reality?

Reply:
Yes, as things are now. As with comments in Slack on not double-counting, I think we'll need a high-level look at making metrics as robust as possible (DLQs, maybe?).